packages/ShowMore/src/ShowMore.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+import PropTypes from "prop-types";
+import truncate from "lodash/truncate";
+import uuid from "uuid/v4";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Button from "@paprika/button";
+import { toggleStyles } from "./ShowMore.styles";
+
+const propTypes = {
+  /** Additional description for "Show more" link. Should be a "topic" that will be appended to "Show more about [topic]". */
+  a11yText: PropTypes.string,
+
+  /* Full content to be revealed. */
+  children: PropTypes.node.isRequired,
+
+  /* Length, in characters, of truncated preview content. */
+  collapsedLength: PropTypes.number,
+};
+
+const defaultProps = {
+  a11yText: null,
+  collapsedLength: 255,
```

Comment:
is this number random?

Reply:
This is the "usual" truncation limit. At least in every case I've implemented a similar function in Projects, this was the arbitrary number.